tokio-codec/src/lib.rs

```diff
@@ -31,4 +32,5 @@ pub use crate::encoder::Encoder;
 pub use crate::framed::{Framed, FramedParts};
 pub use crate::framed_read::FramedRead;
 pub use crate::framed_write::FramedWrite;
+pub use crate::length_delimited::{FrameTooBig, LengthDelimitedCodec};
```

Comment:
I wonder if we should rename this `FrameTooBig` to a better error name similar to the `LinesCodecError`? What do you think?

Reply:
I agree but something like `LengthDelimitedCodecError` is too long IMO.

Reply:
Meh, I think for how often people might use it, I think its fine.

Reply:
Ok, renaming to `LengthDelimitedCodecError` then

Reply:
Done